### ISSUE TITLE
invalid config item -Dio.netty.recycler.maxCapacity.default 

### ISSUE THREAD

**Issue by sunshujie1990** [apache/pulsar#13757]
netty version 4.1.66.Final
Can not find config item io.netty.recycler.maxCapacity.default in io.netty.util.Recycler. I think it's a invalid config.

![image](https://user-images.githubusercontent.com/24857248/149468791-125c9349-ce24-4320-9f3d-e2f406a6df06.png)

**Comment by sunshujie1990** [comment]
@merlimat

**Comment by HQebupt** [comment]
Yes. I agree with you. I had a PR. please help to fix it. Pls help to review it. [Improvement] Recycler should use io.netty.recycler.maxCapacityPerThread instead of io.netty.recycler.maxCapacity.default as maxCapacityPerThread configuration in Netty 4.1.x #13563
